lib/utils.pm

```diff
@@ -2927,4 +2928,27 @@ sub write_sut_file {
     assert_script_run("curl -v -o $path $url");
 }
 
+=head2 handle_gnome_memory_ge_4g
+
+  handle_gnome_memory_ge_4g([fatal_flag => $fatal_flag])
+
+Handle gnome test on sle, system's ram must be at least 4GB
+https://progress.opensuse.org/issues/153808.
+
+=cut
+
+sub handle_gnome_memory_ge_4g {
+    my (%args) = @_;
+    $args{fatal_flag} //= 1;
+
+    if (check_var('DESKTOP', 'gnome') && get_var('GNOME_MEMORY_GE_4G', '1') && is_sle('>=15-SP5') && (is_qemu || is_svirt) && (get_var("QEMURAM", 1024) < 4096)) {
+        if ($args{fatal_flag}) {
+            die "DESKTOP=gnome System's ram must be at least 4GB, see poo#153808";
+        }
+        else {
+            record_soft_failure "DESKTOP=gnome System's ram should be at least 4GB, see poo#153808";
+        }
```

Comment:
Suggestion:
```
die 'DESKTOP=gnome System's ram must be at least 4GB, see poo#153808' if ($args{fatal_flag});
record_soft_failure "DESKTOP=gnome System's ram should be at least 4GB, see poo#153808";
```
No need for the `if - else` block since function will kill the test if the fatal flag is set.